set.mm

```diff
@@ -26688,6 +26688,13 @@ generally appear in a single form (either definitional, but more often
 
   ${
     $d x A y $.  $d ph y $.
+    $( One direction of ~ eqab is provable from fewer axioms.  (Contributed by
+       Wolf Lammen, 13-Feb-2025.) $)
+    eqab1r $p |- ( A. x ( x e. A <-> ph ) -> A = { x | ph } ) $=
```

Comment:
```suggestion
    eqabri $p |- ( A. x ( x e. A <-> ph ) -> A = { x | ph } ) $=
```
(and of course, label needs to be changed in its usages)

Reply:
The problem here is that eqabi exists and is the immediate form of eqab.  So how should a forward implication form be consistently named, then?

Reply:
see your answer now

Reply:
Lets wait for a moment to see how others see this.